api/src/pcapi/core/offerers/repository.py

```diff
@@ -280,6 +281,18 @@ def has_digital_venue_with_at_least_one_offer(offerer_id: int) -> bool:
 )
 
 
+def get_by_collective_booking_id(booking_id: int) -> models.Offerer:
+    query = models.Offerer.query.join(models.Venue, models.Offerer.managedVenues)
+    query = query.join(CollectiveOffer, models.Venue.collectiveOffers)
+    query = query.join(CollectiveStock, CollectiveOffer.collectiveStock)
+    query = query.join(CollectiveBooking, CollectiveStock.collectiveBookings)
+    query = query.filter(CollectiveBooking.id == booking_id)
```

Comment:
On devrait pouvoir s'éviter quelques jointures en utilisant `CollectiveBooking.offererId` : 

```suggestion
    query = models.Offerer.query.join(CollectiveBooking, CollectiveBooking.offererId == models.Offerer.id)
    query = query.filter(CollectiveBooking.id == booking_id)
```